confidant/routes/v1.py

```diff
@@ -669,6 +669,10 @@ def _pair_key_conflicts_for_services(_id, credential_keys, services):
     return conflicts
 
 
+def _lowercase_credential_pairs(credential_pairs):
+    return {i.lower(): j for i, j in credential_pairs.items()}
+
```

Comment:
I think we actually removed this deliberately, unless it's breaking something? The original commit references a specific PR which still hasn't been merged yet: 
https://github.com/stripe/confidant/commit/bf261daca4f6dbdc736b78d654ff76eda8658ece
https://github.com/lyft/confidant/issues/103

Reply:
Yeah, my bad, removed in c73a70d